solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -407,25 +492,38 @@ def process_forecast_observations(forecast_observations, filters, data,
                                     for k, v in counts.items())
                 preproc_results = (datamodel.PreprocessingResult(
                     name=VALIDATION_RESULT_TOTAL_STRING,
-                    count=len(data[fxobs.data_object]) - len(obs_ser)), )
+                    count=int(len(data[fxobs.data_object]) - len(obs_ser))), )
```

Comment:
I'd hope that the difference of two ints would always be an int and this call isn't needed

Reply:
:D One would hope. I may have gotten to happy after a frustrating time with `numpy.int64` not being serializable.